conans/client/conf/__init__.py

```diff
@@ -94,6 +94,25 @@
                 exception: [None]
             Visual Studio:
                 <<: *visual_studio
+    nvcc:
+        version: ["6.0", "6.5", 
+                  "7.0", "7.5", 
+                  "8.0", 
+                  "9.0", "9.1", "9.2", 
+                  "10.0", "10.1", "10.2"]
+        base:
+            gcc:
+                <<: *gcc
+            intel:
+                <<: *intel              
+            clang:
+                <<: *clang
+            apple-clang:
+                <<: *apple-clang
+            Visual Studio:
+                <<: *visual_studio
+        libcxx: [None, libcu++]
```

Comment:
After thinking a bit, `nvcc` shoudn’t have a `libcxx` setting. In 10.2 they clam to have released a std library for GPUs, libcu++ but no documentation is available...